editor/test/internal/cache_test.clj

```diff
@@ -13,18 +13,15 @@
 ;; specific language governing permissions and limitations under the License.
 
 (ns internal.cache-test
-  (:require [clojure.test :refer :all]
-            [support.test-support :refer :all]
+  (:require [clojure.core.cache :as cc]
+            [clojure.test :refer :all]
             [internal.cache :refer :all]
             [internal.graph.types :as gt]
-            [internal.system :as is]))
-
-(defn- mock-system [cache-size ch]
-  (is/make-system {:cache-size cache-size}))
+            [support.test-support :refer :all]))
```

Comment:
The `mock-system` function was unused. Every test was using `with-clean-system`.